src/main/java/uk/gov/companieshouse/api/strikeoffobjections/service/impl/EmailService.java

```diff
@@ -23,69 +21,118 @@
+        }
+    }
+
+    private EmailContent constructEmailContent(EmailType emailType,
+                                               String emailAddress,
+                                               Map<String, Object> data) {
+
+        String typeOfEmail = (emailType == EmailType.CUSTOMER)? emailConfig.getSubmittedCustomerEmailType()
+                : emailConfig.getSubmittedDissolutionTeamEmailType();
+
+        return new EmailContent.Builder()
+                .withOriginatingAppId(emailConfig.getOriginatingAppId())
+                .withCreatedAt(dateTimeSupplier.get())
+                .withMessageType(typeOfEmail)
+                .withMessageId(UUID.randomUUID().toString())
+                .withEmailAddress(emailAddress)
+                .withData(data)
+                .build();
+    }
+
+    private Map<String, Object> constructEmailDataMap(String companyName, Objection objection, String email) {
         Map<String, Object> data = new HashMap<>();
 
         LocalDate submittedOn = objection.getCreatedOn().toLocalDate();
 
-        String subject = emailSubject.replace("{{ COMPANY_NUMBER }}", companyNumber);
+        String emailSubject = emailConfig.getEmailSubject();
+        String subject = emailSubject.replace("{{ COMPANY_NUMBER }}", objection.getCompanyNumber());
         data.put("subject", subject);
-        data.put("to", emailAddress);
         data.put("date", FormatUtils.formatDate(submittedOn));
         data.put("objection_id", objection.getId());
+        data.put("to", email);
         data.put("company_name", companyName);
-        data.put("company_number", companyNumber);
+        data.put("company_number", objection.getCompanyNumber());
+        data.put("objection_id", objection.getId());
         data.put("reason", objection.getReason());
         data.put("attachments", objection.getAttachments());
+        return data;
+    }
 
-        EmailContent emailContent = new EmailContent.Builder()
-                .withOriginatingAppId(originatingAppId)
-                .withCreatedAt(dateTimeSupplier.get())
-                .withMessageType(submittedCustomerEmailType)
-                .withMessageId(UUID.randomUUID().toString())
-                .withEmailAddress(emailAddress)
-                .withData(data)
-                .build();
-
-        logger.debugContext(requestId, "Calling Kafka client to send email");
-        kafkaEmailClient.sendEmailToKafka(emailContent);
-        logger.debugContext(requestId, "Successfully called Kafka client");
+    protected String[] getDissolutionTeamRecipients(String jurisdiction) {
+        switch(jurisdiction) {
+            case "england":
+                return emailConfig.getEmailRecipientsCardiff().split(",");
+            case "wales":
+                return emailConfig.getEmailRecipientsCardiff().split(",");
+            case "england-wales":
+                return emailConfig.getEmailRecipientsCardiff().split(",");
```

Comment:
Appreciate @ICoxCH 's comment above and I haven't seen original value of code, but not using something like:

case "england": case "wales": case "england-wales":
   return emailConfig.getEmailRecipientsCardiff().split(",");
...

kind of negates the use of switch/case - it's intended to improve readability and cut down on repeated code.

Reply:
Yeh I know - I had a comment above about this saying the opposite - I think there are multiple points of view here.